tests/epilogue.test.js

```diff
@@ -34,6 +34,22 @@ describe('Epilogue', function() {
     done();
   });
 
+  it('should extend .resource options if they are provided on initialize', function() {
+    var resourceOptions = { pagination: false };
+
+    epilogue.initialize({
+      app: {},
+      sequelize: new Sequelize('main', null, null, {
+        dialect: 'sqlite',
+        storage: ':memory:',
+        logging: (process.env.SEQ_LOG ? console.log : false)
+      }),
+      resource: resourceOptions
```

Comment:
Honestly I don't know, but I don't feel right giving non-semantic definitions on things we don't know if will change. I've understood why you prefer so, but with `defaults` key I would see something more like: `defaults: { resource: { /* ... */ } }` (In fact I was going to implement that way but I've changed my mind because sequelize is using `{ define: {  /* defaults for define */ } }`).

My motto is always to avoid future BC by having good implementations. Even if this is only creating `resources`, it doesn't mean we could introduce another kind of bindings later.

But it is your decision, tell me what you think about, I will write it. :)